aws/b2b_personalization/_env/tenant/authentication.hcl

```
@@ -0,0 +1,20 @@
+locals {
+  env_vars      = read_terragrunt_config(find_in_parent_folders("env.hcl"))
+  tenant_config = read_terragrunt_config(find_in_parent_folders("tenant_config.hcl"))
+}
+
+terraform {
+  source = "../../../../modules/lambda//authentication"
+}
+
+inputs = {
+  tenant = local.tenant_config.locals.name
+  tags = merge(
```

I applied the `hclfmt` and it shows that it is good, but shouldn't the format be like this:

```suggestion
  tenant = local.tenant_config.locals.name
  tags    = merge(
```